VaccinationSystem/VaccinationSystem/Data/Migrations/20220326221422_VaccinationYouAreAVisitNow.cs

```diff
@@ -0,0 +1,224 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace VaccinationSystem.Data.Migrations
+{
+    public partial class VaccinationYouAreAVisitNow : Migration
```

Comment:
To niezła nazwa

Reply:
![33jeh2](https://user-images.githubusercontent.com/92259475/160354331-1cd41ec2-a526-49a5-a370-440a15251db5.jpg)